### ISSUE TITLE
Live premiere(/edited?) segments submissions not accurately represented.

### ISSUE THREAD

**Issue by mirror176** [ajayyy/SponsorBlock#523]
Submitted edits on content listed as live while watching it (didn't tune in at its start but started from its beginning) with several issues. While watching, the timebar markings failed to hold to the segments based on thumbnail preview; suspect it is from a dynamically lengthening timeline. After airing it went to an edited form where at least the intro countdown timer was gone and no segments matched anymore; have not tested this on videos content creators edit after it had been up for some time but suspect there may be a similar issue.
  It would be handy to either have live submissions blocked (haven't tested if they work to skip correctly during the live presentation for other viewers), or an easy way to report segment offsets so they could be slid. I think I have had 1-2 other submissions where this could have had an impact but didn't have a chance to review once submitted so hopefully the voting system fixed it. I fixed my most recent attempt with downvote on bad skips and resubmission but not sure I got em all and definitely didn't get em as precise. Not sure how it impacts other addon users as I thought there was an effort to make a single user not receive skips they submitted as bad anymore but leave it in the pool for others. Last video was Ci_eOrWB8b4 (warning, contains non-mainstream political content).

**Comment by ajayyy** [comment]
Live submissions are fine, that is just a visual issue that unfortunately doesn't have any good solution.

Premieres are two minutes off though

**Comment by mirror176** [comment]
If premiere is detected then maybe it could autosubtract if the time adjustment is always 2 minutes. Does sponsorblock record full video time to watch for other edits in the future?

**Comment by ajayyy** [comment]
The problem is that it seems premiers aren't exactly two minutes in my testing. They are 1:58, or something like that. It does not.

**Comment by Xaapyks** [comment]
Hello,

I just noticed the same issues with premiere videos:
1st issue : visual issue where the segment is displayed with some offset on the play bar, this is minor, and it does skip the segment at the intended time (where it was created) and not the displayed time.
2nd issue : This one is major, once the video is no longer premiere, the segments are misplaced, skipping a wrong part of the video. So to be clear, the segments that were working during premiere replay no longer match the same video once out of premiere mode.

I downvoted my now misplaced segment and recreated it on the non-premiere video, but I am unsure if I could have just really deleted it instead (but did not find how)
